.travis.yml

```diff
@@ -396,3 +396,15 @@ after_success:
         echo "deploy to maven snapshot repository is finished";
     fi
     sleep 5s
+  - |
+    fail=0
+    if [ -z "$(ls -A .ci-temp)" ]; then
+        echo "Empty .ci-temp/ validation did not find any warnings."
+    else
+        echo "Directory .ci-temp/ is not empty. Verification failed."
+        echo "Contents of .ci-temp/:"
+        fail=1
+    fi
+    ls -A .ci-temp --color=auto
+    exit $fail
+    sleep 5s
```

Comment:
we need to sleep before exit. 
To let output successfully flushed in log file before container closure (base on experience, command before exit usually excluded from logs). I always wanted to find time to report this to Travis or find existing issue on this, as it might have already more elegant solution.

Reply:
Ok! I didn't know why all the sleeps were in there, I just figured it was for appearance/ human readability.  Fixed.

Reply:
Do you notice this issue only when nonzero exit status occurs, or always?  Also, I didn't use `set -e`, should I update to include `set -e` option?

Reply:
Please add `set -e` it is always good to fail quick in CI even on minors. It is much better than miss leak of violations or errors.

Reply:
@romani This looks like what you're describing: https://travis-ci.community/t/log-are-truncated-when-my-script-exits-abnormally/5878/4 Note that the OP is stating that this happens when script exits normally or with error.